### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by wathen**
Adding github actions to install

1. `fabm-ersem`
2. `gotm-fabm-ersem`

### CODE REVIEW

File: .github/workflows/ersem.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          run: sudo apt-get install gfortran libnetcdff-dev
+        - name: Cloning FABM and GOTM
+          run: |
+              git clone https://github.com/fabm-model/fabm.git
+              git clone https://github.com/gotm-model/code.git gotm
+              cd gotm && git submodule update --init --recursive && cd ..
+        - name: Building GOTM-FABM-ERSEM
+          run: |
+              mkdir build && cd build
+              cmake ../gotm/src -DFABM_BASE=../fabm -DFABM_ERSEM_BASE=.. -DCMAKE_Fortran_COMPILER=/usr/bin/gfortran
```
Comment (jornbr) [inline .github/workflows/ersem.yml:51]: Better change `../gotm/src` to `../gotm`? The former is deprecated and will break someday. There is a warning somewhere among all the cmake output.

Comment (wathen) [inline .github/workflows/ersem.yml:51]:
```suggestion
              cmake ../gotm -DFABM_BASE=../fabm -DFABM_ERSEM_BASE=.. -DCMAKE_Fortran_COMPILER=/usr/bin/gfortran
```

Comment (wathen) [reply .github/workflows/ersem.yml]: Corrected

File: .github/workflows/ersem.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+              cmake ../fabm/src/drivers/python -DFABM_ERSEM_BASE=..
+              make install
+        - name: Check installation
+          run: python -c "import fabm"
+  build-gotm-fabm-ersem:
+    runs-on: ubuntu-latest
+    steps:
+        - name: Checkout branch
+          uses: actions/checkout@v2
+        - name: Setup python
```
Comment (jornbr) [inline .github/workflows/ersem.yml:37]: I think this job can do without python?

Comment (wathen) [reply .github/workflows/ersem.yml:37]: Ahh yes, the `gotm-fabm-ersem` does not need `python` I'm guessing?

Comment (wathen) [reply .github/workflows/ersem.yml:37]: Do you think the same would be try for `gfortran`? It was using `f95` before

Comment (jornbr) [reply .github/workflows/ersem.yml:37]: yes gotm-fabm-ersem does not require python. On the systems I'm familiar with, f95 is an alias for gfortran. If so, that would be fine as long as the version is 4.9 or higher. Did cmake report more details about the compiler (e.g. version)?

Comment (wathen) [reply .github/workflows/ersem.yml]: I removed `gfortran` and it seemed to be fine. You can see the `cmake` report but not the full log file. I was hoping that there would be a debug mode (similar to the one in travis) which one can use to debug the yaml file, I have not found this yet...

**Pull request merged**